ultimatejobweb/models.py

```diff
@@ -2,9 +2,24 @@
 from django.utils import timezone
 
 
-class homepage(models.Model):
-    author = models.CharField(max_length=200)
-    text = models.TextField()
+class user(models.Model):
```

Comment:
django comes with a user model for authentication. You can either completely extend it or create a second OneToOne  table that contains the additional fields that you want (my personal preference).
See: https://docs.djangoproject.com/en/3.1/topics/auth/customizing/#extending-the-existing-user-model
https://docs.djangoproject.com/en/3.1/topics/auth/customizing/#substituting-a-custom-user-model
thisarticle might be a simpler explanation:
https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html